packages/EasyCore/src/Bridge/Symfony/Serializer/TrimStringsDenormalizer.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Symfony\Serializer;
+
+use EonX\EasyCore\Helpers\StringsTrimmerInterface;
+use InvalidArgumentException;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use Symfony\Component\Serializer\SerializerAwareInterface;
+use Symfony\Component\Serializer\SerializerInterface;
+
+/**
+ * @SuppressWarnings(PHPMD.UnusedFormalParameter) Method signatures are defined by parent
+ */
+final class TrimStringsDenormalizer implements DenormalizerInterface, NormalizerInterface, SerializerAwareInterface
```

Comment:
If we decided to implement both, `DenormalizerInterface` and `NormalizerInterface`, then it would be better to rename this class to `TrimStringsNormalizer`.

Reply:
Done.